src/openforms/js/components/admin/forms/ReactSelect.js

```diff
@@ -63,9 +87,15 @@ const Select = ({name, options, ...props}) => {
   );
 };
 
-Select.propTypes = {
+SelectWithoutFormik.propTypes = {
+  name: PropTypes.string.isRequired,
+  options: PropTypes.arrayOf(PropTypes.object),
+};
```

Comment:
I think we should make `value` and `onChange` required props

Reply:
Ah yes, will do!

Reply:
Mhhh I don't think value can be a required prop. Seeing how it can be null/undefined